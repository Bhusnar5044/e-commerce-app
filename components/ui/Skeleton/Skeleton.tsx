import { FC, memo } from 'react';

import { cn } from '@utils';

export const Skeleton: FC = memo(({ className, ...props }: React.HTMLAttributes<HTMLDivElement>) => {
    return <div className={cn('animate-pulse rounded-md bg-neutral-200', className)} {...props} />;
});

Skeleton.displayName = 'Skeleton';
