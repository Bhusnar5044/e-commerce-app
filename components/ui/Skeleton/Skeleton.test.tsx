import { render } from '@test-utils';
import { Skeleton } from './Skeleton';

describe('Skeleton', () => {
    it('should render properly', () => {
        const { container } = render(<Skeleton />);
        expect(container).toMatchSnapshot();
    });
});
