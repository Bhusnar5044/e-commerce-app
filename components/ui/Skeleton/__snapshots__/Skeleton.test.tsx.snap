// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton should render properly 1`] = `
<div>
  <div
    class="animate-pulse rounded-md bg-neutral-200"
  />
</div>
`;
